packages/components/pagination/src/index.js

```diff
@@ -0,0 +1,3 @@
+export { default } from './pagination';
```

Comment:
Early feedback. I would split this into multiple components and expose them like

* `TablePagination` (the full combined components)
<img width="934" alt="image" src="https://user-images.githubusercontent.com/1110551/102871888-466b5600-443f-11eb-85d7-b1a5ac34cf14.png">

* `PaginationInput` (only the left/right page part)
<img width="201" alt="image" src="https://user-images.githubusercontent.com/1110551/102871776-22a81000-443f-11eb-8c2c-2f648e5c1f37.png">

* `PerPageInput` (only the selector part)
<img width="197" alt="image" src="https://user-images.githubusercontent.com/1110551/102872413-ee811f00-443f-11eb-9dda-f604c2286aa3.png">

---

Naming inspiration from [material ui](https://material-ui.com/components/pagination/#table-pagination)

Reply:
Splitting makes sense 👍 

I was more inclined towards `Pagination` instead of `TablePagination` because I can see this component being used for other things than just tables.